src/linuxX64Main/kotlin/mu/internal/ConsoleAppender.kt

```diff
@@ -0,0 +1,43 @@
+package mu.internal
+
+import kotlinx.cinterop.CPointer
+import mu.KotlinLoggingConfiguration
+import platform.posix.FILE
+import platform.posix.fflush
+import platform.posix.fprintf
+
+object ConsoleAppender : IAppender {
+    private val STDOUT = platform.posix.fdopen(1, "w")
+    private val STDERR = platform.posix.fdopen(2, "w")
```

Comment:
Instead of explicitly opening a descriptor, can you use `platform.posix.stderr` and `platform.posix.stdout`?

Reply:
Switched